### PR TITLE
[docker] fixing the "backup_dir hasnt got permissions" error

### PR DESCRIPTION
**Pull request by imtherealF1**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ services:
     restart: always
     user: postgres:postgres
     volumes:
-      - /var/opt/pgbackups:/backups
```
Comment (ltzmax) [inline docker-compose.yml:70]: What OS are you actually getting the issue from?

Comment (imtherealF1) [reply docker-compose.yml:70]: "backup_dir redirects to a file without permissions" was popping up on my docker logs constantly both windows and linux, and this fixed it

Comment (ltzmax) [reply docker-compose.yml:70]: Well this has never popped up to me on linux at all, the best fix is to give the location permission.

Comment (laggron42) [reply docker-compose.yml:70]: It's an issue for Windows, and it's going to make it easier to sort tbh

**Review by laggron42 — APPROVED** [review]

**Pull request merged**